docs/release_notes/release_notes-0.15.0.rst

```diff
@@ -0,0 +1,31 @@
+Modin 0.15.0
+
+Key Features and Updates
+------------------------
+
+* Stability and Bugfixes
+  *
+* Performance enhancements
+  *
+* Benchmarking enhancements
+  *
+* Refactor Codebase
+  *
+* Pandas API implementations and improvements
+  *
+* OmniSci enhancements
+  *
+* XGBoost enhancements
+  *
+* Developer API enhancements
+  *
+* Update testing suite
+  *
+* Documentation improvements
+  * DOCS-#4350: Add release notes template for 0.15.0 (#4351)
```

Comment:
I do not think we need the template to be a part of the release notes 😄

Reply:
Yes, I agree. I was thinking about to not include this but added eventually 😄 Removed.